src/effects/native/flangereffect.cpp

```diff
@@ -27,18 +27,17 @@ EffectManifest FlangerEffect::getManifest() {
         "A simple modulation effect, created by taking the input signal "
         "and mixing it with a delayed, pitch modulated copy of itself."));
 
-    EffectManifestParameter* period = manifest.addParameter();
-    period->setId("period");
-    period->setName(QObject::tr("Period"));
-    period->setDescription(QObject::tr("Controls the period of the LFO (low frequency oscillator)\n"
-        "1/4 - 20 beats rounded to 1/2 beat if tempo is detected (decks and samplers) \n"
-        "0.05 - 20 seconds if no tempo is detected (mic & aux inputs, master mix)"));
-    period->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
-    period->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
-    period->setUnitsHint(EffectManifestParameter::UnitsHint::BEATS);
-    period->setMinimum(0.00);
-    period->setMaximum(20.00);
-    period->setDefault(1.00);
+    EffectManifestParameter* speed = manifest.addParameter();
+    speed->setId("speed");
+    speed->setName(QObject::tr("Speed"));
+    speed->setDescription(QObject::tr("Controls the speed of the LFO (low frequency oscillator)\n"
+        "32 - 1/4 beats rounded to 1/2 beat per lfo cycle if tempo is detected (decks and samplers) \n"
+        "1/32 - 4 Hz if no tempo is detected (mic & aux inputs, master mix)"));
+    speed->setControlHint(EffectManifestParameter::ControlHint::KNOB_LOGARITHMIC_INVERSE);
```

Comment:
I don't understand why this is inverted or why the parameter was renamed. I'm also not sure this should be logarithmic. The user cannot know the value of a logarithmic knob just by looking at the position. @sohet, what do you think?

If we do keep the logarithmic behavior, then the rounding to fractions doesn't make much sense IMO. I think it would be more appropriate to round to powers of 2 like the old behavior of the AutoPan Period parameter.

Reply:
Turning the "speed" parameter knob right, the sound grows aggresive as other parameters (ex. "regen" of this effect). It seems natural for me.

It is difficult to control speed or period when the value is small with linear scale. So if the small value region is important, logarithmic scale is worth considering. It is desirable that the knob movement from 1/8 to 1/4 beat is equal to the one from 8 to 16 beat.

Powers of 2 rounding always locks LFO to beats, so can not create tension as @christophski fears about.